src/components/hoc-helpers/withHeaderName.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+
+const withHeaderName = (View) => {
+  return class extends Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+          controlHeader: '',
+          setControlHeader: this.setControlHeader
+        };
+    }
+  
+    setControlHeader = (event, controlHeader) => {
```

Comment:
I haven't found any places where you use this method

Reply:
Oops, that's totally redundant then. I should remove that.

Reply:
think of all the component logic, if you remove the method it might happen that you don't need the state

Reply:
I've removed the whole component. Done.